docs/en/03-benchmark/supported_models.md

```diff
@@ -23,6 +23,7 @@ The table below lists the models that are guaranteed to be exportable to other b
 | [DETR](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/detr)                                 | MMDetection      |      N      |      Y      |    Y     |  N   |   ?   |    N     |   N    |  N   |
 | [CenterNet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/centernet)                       | MMDetection      |      N      |      Y      |    Y     |  N   |   ?   |    N     |   N    |  N   |
 | [SOLO](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/solo)                                 | MMDetection      |      N      |      N      |    N     |  N   |   N   |    Y     |   N    |  N   |
+| [SOLOv2](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/solov2)                             | MMDetection      |      N      |      N      |    N     |  N   |   N   |    Y     |   N    |  N   |
```

Comment:
Hi could you add the test config in yml like solo?
https://github.com/open-mmlab/mmdeploy/blob/6c26e887d4300c3afb57a0900c6ad53798d9dfda/tests/regression/mmdet.yml#L348

Reply:
pls. add onnxruntime pipeline as well.

Reply:
@Li-Hongda Hi, sorry for the late reply. this comment is not resolved.



> Hi could you add the test config in yml like solo?
> 
> https://github.com/open-mmlab/mmdeploy/blob/6c26e887d4300c3afb57a0900c6ad53798d9dfda/tests/regression/mmdet.yml#L348

Reply:
Sorry, I didn't notice that.